src/t8_eclass.c

```diff
@@ -39,31 +39,31 @@ const int t8_face_vertex_to_tree_vertex[T8_ECLASS_COUNT][T8_ECLASS_MAX_FACES][T8
   { { 0, 2, 4 }, { 1, 3, 4 }, { 0, 1, 4 }, { 2, 3, 4 }, { 0, 1, 2, 3 } }                              /* pyramid */
 };
 
-/* TODO: tet, prism, pyramid */
+/* TODO: tet, pyramid */
 const int t8_face_edge_to_tree_edge[T8_ECLASS_COUNT][T8_ECLASS_MAX_FACES][T8_ECLASS_MAX_EDGES_2D] = {
   { { -1 } },                                                                                             /* vertex */
   { { 0 } },                                                                                              /* line */
   { { 0 }, { 1 }, { 2 }, { 3 } },                                                                         /* quad */
   { { 0 }, { 1 }, { 2 } },                                                                                /* triangle */
   { { 8, 10, 4, 6 }, { 9, 11, 5, 7 }, { 8, 9, 0, 2 }, { 10, 11, 1, 3 }, { 4, 5, 0, 1 }, { 6, 7, 2, 3 } }, /* hex */
   { { -1 } },                                                                                             /* tet */
-  { { -1 } },                                                                                             /* prism */
+  { { 0, 3, 6, 7 }, { 1, 4, 7, 8 }, { 2, 5, 6, 8 }, { 0, 1, 2 }, { 3, 4, 5 } },                           /* prism */
```

Comment:
For the triangular surfaces the edges are ordered counterclockwise, but for the quadrilateral surfaces they aren't. I like the counterclockwise ordering, we should be consistent about it.